WebDev-3.html

```diff
@@ -97,6 +106,45 @@ <h5 class="right">GIS4533 - Fall 2020 - Nate Ickert</h5>
     // Create variable to hold map element, give initial settings to map
     var map = L.map('map',{ center: [46.7867, -92.1005], zoom: 12});
     // Add OpenStreetMap tile layer to map element
+
+        
+         var Stadia_AlidadeSmoothDark = L.tileLayer('https://tiles.stadiamaps.com/tiles/alidade_smooth_dark/{z}/{x}/{y}{r}.png', {
+	maxZoom: 20,
+	attribution: '&copy; <a href="https://stadiamaps.com/">Stadia Maps</a>, &copy; <a href="https://openmaptiles.org/">OpenMapTiles</a> &copy; <a href="http://openstreetmap.org">OpenStreetMap</a> contributors',
+             apikey: 'd8ebd0005b0d499b9953b6859696b681',
+}).addTo(map);
+        
+         
+    
+        omnivore.csv('airports.csv').addTo(map);  
+       
+        var EngerDataPoint = L.marker([46.7761, -92.1250]).addTo(map);
+        
+        EngerDataPoint.bindPopup("<h3>Enger Tower</h3><p>Duluth, Mn<br>Beautiful view on the hill.</p>");
+        
+        var AmsoilDataPoint = L.marker([46.7804, -92.0997]).addTo(map);
+        
+        AmsoilDataPoint.bindPopup("Duluth, MN.");
+        
+        var LiftBridgeDataPoint = L.marker([46.7797, -92.0935]).addTo(map);
+        
+        LiftBridgeDataPoint.bindPopup("Duluth,MN.");
+        
+        var popup = L.popup();
+
+// Write function to set Properties of the Popup
+function onMapClick(e) {
+    popup
+        .setLatLng(e.latlng)
+        .setContent("You clicked the map at " + e.latlng.toString())
+        .openOn(map);
+}
+
+// Listen for a click event on the Map element
+map.on('click', onMapClick);
+   
+
+=======
```

Comment:
My guess is that this resulted from a merge conflict at some point.
This text that is inserted into your code can be annoying if you don't know to look for it.